### DISCUSSION TITLE
Repo Settings Access for Maintainers

### DISCUSSION THREAD

**Discussion by wynquinox** [grpc-requests/grpc_requests#51] [General]
@wesky93 - what repo setting access can we set up for maintainers?
Given that we're seeing folks like @artificial-aidan making some seriously good contributions to the repo - I think it'd be a good idea to get some branch protection rules for a minimum number of approving reviews set up, and to allow one or two core maintainers to be able to take care of the operations of the repo to allow for continuance of operations if you or I get busy or out of communication for a long period of time.

**Comment by wesky93** [comment]
sound good!
Why don't we invite the people you mentioned to become maintainers of the repo?

**Comment by wesky93** [comment]
Public repos don't seem to have a designated maintainer. 
I invited him as a collaborator.

 @artificial-aidan wellcom!

**Comment by aidandj** [comment]
Thanks! I just wrapped up implementation using this library,so hopefully no changes anytime soon. Mention me if something I contributed acts up.

**Comment by wynquinox** [comment]
Thanks for the contributions @artificial-aidan - will definitely let you know if anything comes up.

@wesky93 - sounds like I'll just have to rely on you to update internal things like if we need to make sure pypi has a 2FA account associated key, or if we wanted to update the code coverage tokens. I just just open issues around stuff like that and assign it to you.
